integration/kubernetes/multiMaster-embeddedJournal/alluxio-configmap.yaml.template

```diff
@@ -28,16 +28,16 @@ metadata:
     heritage: Tiller
 data:
   ALLUXIO_JAVA_OPTS: |-
-     -Dalluxio.master.journal.type=EMBEDDED -Dalluxio.master.journal.folder=/journal -Dalluxio.master.embedded.journal.addresses=alluxio-master-0:19200,alluxio-master-1:19200,alluxio-master-2:19200,  -Dalluxio.security.stale.channel.purge.interval=365d 
```

Comment:
we might still need the leading / trailing space, until we fix the issue with appending common java opts in libexec. alternatively, you could fix that issue

Reply:
There was an extra space and is now removed. The leading/trailing spaces are still aligned. I ran through the tests and they are all good.